.travis.yml

```diff
@@ -18,32 +24,41 @@ addons: &apt_64
     - libpulse-dev
 env:
   # default jobs
-  - ARCH=x86_64
-  - ARCH=x86_64 CARGOOPTS="--features pulse-dlopen"
+  -
+  - FEATURES=pulse-dlopen
 matrix:
   include:
   # Add in 32-bit builds
   - rust: 1.15.1
-    env: ARCH=i686
+    env:
+      - TARGET=i686-unknown-linux-gnu
     addons: &apt_32
       apt:
         packages:
         - gcc-multilib
         - libpulse-dev:i386
   - rust: 1.15.1
-    env: ARCH=i686 CARGOOPTS="--features pulse-dlopen"
+    env:
+      - TARGET=i686-unknown-linux-gnu
+      - FEATURES=pulse-dlopen
     addons: *apt_32
   - rust: stable
-    env: ARCH=i686
+    env:
+      - TARGET=i686-unknown-linux-gnu
     addons: *apt_32
   - rust: stable
-    env: ARCH=i686 CARGOOPTS="--features pulse-dlopen"
+    env:
+      - TARGET=i686-unknown-linux-gnu
+      - FEATURES=pulse-dlopen
     addons: *apt_32
   - rust: nightly
-    env: ARCH=i686
+    env:
+      - TARGET=i686-unknown-linux-gnu
     addons: *apt_32
   - rust: nightly
-    env: ARCH=i686 CARGOOPTS="--features pulse-dlopen"
+    env:
+      - TARGET=i686-unknown-linux-gnu
+        - FEATURES=pulse-dlopen
```

Comment:
Indentation.  I set up all the - items with the - aligned to the parent indentation, not indented by 2 spaces.  For example, see:
```yaml
apt:
  packages:
  - gcc-multilib
```